doc/user/content/sql/create-sink.md

```diff
@@ -178,7 +178,9 @@ When you create a sink, you must:
 * Enable the `reuse_topic` switch.
 * Specify a [consistency topic](#consistency-metadata) to store the information that Materialize will use to identify the last completed write. The names of the sink topic and the sink consistency topic must be unique across all sinks in the system.
 
-The sink consistency topic cannot be written to by any other process, and both the sink topic and the sink consistency topic need to be set to infinite data retention.
+The sink consistency topic cannot be written to by any other process, including another Materialize instance or another sink.
+
+With `reuse_topic` enabled, the schema for topic naming mentioned above is ignored. Instead, the topic name specified in the sink definition is used as is.
```

Comment:
Instead of putting this here, let's just make the bullet point in "Kafka sinks" "If you haven't specified the name for [topic reuse](#topic-reuse), Materialize names the new topic using the format below."

The schema from naming is far away from this section. It's also generally a bad idea to use directional instructions for tech writing because they can become inaccurate if you reuse content in different places.

(You're also going to get a merge conflict, I think -- the doc was updated after this version.)

Reply:
Thanks, that's a very good point! I moved it to that section as a `**Note:** ...`, you think that's alright?